site/docs/releases.md

```diff
@@ -88,6 +88,20 @@ The 1.8.1 release contains bug fixes and fixes to LICENSE/NOTICE files. For full
 
 ## Past releases
 
+### 1.7.2 release
```

Comment:
nit: Maybe move this below 1.8.0 because users should see the latest version first for download?